app/models/user.rb

```diff
@@ -11,4 +12,22 @@ def self.digest string
     cost = ActiveModel::SecurePassword.min_cost ? BCrypt::Engine::MIN_COST : BCrypt::Engine.cost
     BCrypt::Password.create string, cost: cost
   end
+
+  def self.new_token
```

Comment:
self này là gì e?

Reply:
Dạ nó cũng giống như "this" ở các ngôn ngữ khác. có thể gọi để thay thế cho tên của lớp và được định nghĩa như 1 lớp phương thức.